### DISCUSSION TITLE
Reset form dirty state without calling reset

### DISCUSSION THREAD

**Discussion by bufordtaylor** [react-hook-form/react-hook-form#1682] [General] [answered]
I am using react-froala-wysiwyg in one of the form fields. I register the component manually to make it work via useEffect

```
  useEffect(() => {
    register({ name: "description" }, { required: true })
    return () => unregister("description")
  }, [register])
```

When I save, I would like to clean the dirty state. In the docs and in the github issues, it seems the common approach is to `reset(getValues())`. However, calling reset causes the froala wysiwig to vanish.

![CleanShot 2020-05-23 at 18 26 54](https://user-images.githubusercontent.com/214319/82741762-6ad54780-9d23-11ea-87b5-38a9f997eaa6.gif)

Is there a way to reset the dirty state without calling reset on all the form fields?

**Comment by bufordtaylor** [comment, marked as answer]
Interesting follow-up here. If I reset the value to anything other than the current value, it doesn't vanish

```
reset({...getValues(), description: getValues()['description']+' '}}
```

**Reply by bluebill1049** [reply]
that's because you are using custom register if you are using Controller. This will work.
